client/verta/tests/unit_tests/deployment/test_build.py

```diff
@@ -2,12 +2,12 @@
 
 from typing import Any, Dict
 
-from hypothesis import given, HealthCheck, settings
 import hypothesis.strategies as st
+import pytest
+from hypothesis import given, HealthCheck, settings
```

Comment:
Would you mind keeping the two `hypothesis` lines grouped together?

Reply:
I ran PyCharm's `Optimize Imports` feature and this seems to be what it came up with.  Not sure why that would be the outcome, but I'll happily rearrange to keep like elements together.

Reply:
Ah, I didn't realize it was from a tool.

Well...I guess that's fine? I have no idea why a tool would want to split up a package like this but it's not a big deal 🤷

Reply:
Done